### PR TITLE
Include binaries and upload to testpypi

### PR DESCRIPTION
**Pull request by paddyroddy**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -163,6 +163,7 @@ dynamic.dependencies.file = "requirements.txt"
 packages.find = {}
 
 [tool.setuptools_scm]
+local_scheme = "no-local-version"
```
Comment (paddyroddy) [inline pyproject.toml:166]: Useful for local loading to PyPI

Comment (paddyroddy) [reply pyproject.toml:166]: i.e. on the same tag

Comment (p-j-smith) [reply pyproject.toml:166]: just wondering what this setting does? Does it change the version installed locally?

**Review by p-j-smith — APPROVED** [review]
awesome! let's get a release out shortly

**Comment by paddyroddy** [comment]
https://github.com/quantumjot/btrack/pull/294#discussion_r1157435704

If you run `python -m twine upload --repository testpypi dist/*` locally, you'll be blocked from `TestPyPI` if you don't have that bit.

**Pull request merged**